platform-coop-toolkit/coop-map-directory-index
# feat: add affiliation url (resolves #86)

## Description

Adds field for affiliation URLs to user profiles (used for roles with an institutional/organizational affiliation, e.g. Researcher).

## Steps to test

1. Check out branch.
2. Run migrations for accounts.

**Expected behavior:** "Affiliation url" field appears in account admin.

## Additional information

Not applicable.

## Related issues

- Requires #88.
- Blocker for #84.
- Resolves #85.
